src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -0,0 +1,185 @@
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.infrastructure.app.commands
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Text;
+    using System.Xml;
+    using attributes;
+    using commandline;
+    using configuration;
+    using filesystem;
+    using infrastructure.commands;
+    using logging;
+    using services;
+    using tolerance;
+
+    [CommandFor("export", "exports list of currently installed packages")]
+    public class ChocolateyExportCommand : ICommand
+    {
+        private readonly INugetService _nugetService;
+        private readonly IFileSystem _fileSystem;
+
+        public ChocolateyExportCommand(INugetService nugetService, IFileSystem fileSystem)
+        {
+            _nugetService = nugetService;
+            _fileSystem = fileSystem;
+        }
+
+        public void configure_argument_parser(OptionSet optionSet, ChocolateyConfiguration configuration)
+        {
+            optionSet
+                .Add("o=|output-file-path=",
+                     "Output File Path - the path to where the list of currently installed packages should be saved. Defaults to packages.config.",
+                     option => configuration.ExportCommand.OutputFilePath = option.remove_surrounding_quotes())
+                .Add("include-version-numbers",
+                     "Include Version Numbers - controls whether or not version numbers for each package appear in generated file.  Defaults to false.",
+                     option => configuration.ExportCommand.IncludeVersionNumbers = option != null)
+                ;
+        }
+
+        public void handle_additional_argument_parsing(IList<string> unparsedArguments, ChocolateyConfiguration configuration)
+        {
+            if (unparsedArguments.Count > 0)
+            {
+                throw new ApplicationException("Please see the help menu for the export command");
+            }
+        }
+
+        public void handle_validation(ChocolateyConfiguration configuration)
+        {
+            // Currently, no additional validation is required.
+        }
+
+        public void help_message(ChocolateyConfiguration configuration)
+        {
+            this.Log().Info(ChocolateyLoggers.Important, "Export Command");
+            this.Log().Info(@"
+Export all currently installed packages to a file.
+
+This is especially helpful when re-building a machine that was created
+using Chocolatey.  Export all packages to a file, and then re-install
+those packages onto new machine using `choco install packages.config`.

```

Comment:
typo? - https://github.com/gep13/choco/pull/1

Reply:
@mwallner agreed.  Minor tweak required on your PR, and then I can get it pulled in.